apps/src/templates/account/AccountBanner.tsx

```diff
@@ -10,11 +10,14 @@ import styles from './account-components.module.scss';
 const AccountBanner: React.FunctionComponent<{
   heading: string;
   desc: string;
-}> = ({heading, desc}) => (
+  showLogo: boolean;
+}> = ({heading, desc, showLogo}) => (
   <div className={styles.bannerContainer}>
-    <div className={styles.iconContainer}>
-      <img src={codeLogo} alt={i18n.codeLogo()} />
-    </div>
+    {showLogo && (
```

Comment:
See this [comment](https://github.com/code-dot-org/code-dot-org/pull/60209/files#r1705856571) and this [comment](https://github.com/code-dot-org/code-dot-org/pull/60209/files#r1705856983).